src/Event/ProophActionEventEmitter.php

```diff
@@ -42,7 +42,7 @@ public function __construct(array $availableEventNames = [])
      *
      * @return ActionEvent that can be triggered by the ActionEventEmitter
      */
-    public function getNewActionEvent(string $name = null, $target = null, $params = null): ActionEvent
+    public function getNewActionEvent(?string $name = null, $target = null, $params = null): ActionEvent
```

Comment:
same for target and params, no?

Reply:
I think that non-typed parameters are already defined as mixed, which already contains null.

After doing some research I found this: (Read the section Dropping parameter type)

https://php.watch/versions/8.4/implicitly-marking-parameter-type-nullable-deprecated

Reply:
sorry my bad